OpenRA.Game/Widgets/Widget.cs

```diff
@@ -347,7 +347,7 @@ void ForceYieldKeyboardFocus()
 				Ui.KeyboardFocusWidget = null;
 		}
 
-		public virtual string GetCursor(int2 pos) { return "default"; }
+		public virtual string GetCursor(int2 pos) { return ChromeMetrics.Get<string>("DefaultCursor"); }
```

Comment:
This still needs to be cached (I did mention this in my last review).

Reply:
Fixed.